### PR TITLE
2019 immersive web tech radar

### PR DESCRIPTION
**Pull request by TrevorFSmith**
This PR probably won't be merged but it's a handy way to gather feedback on the list and order for a technical radar, as described in [this email to the public list](https://lists.w3.org/Archives/Public/public-immersive-web/2019Apr/0004.html)

The goal is to gather and roughly order by time the high-level aspects of the immersive web that will need to come together in order for the web to be a performant, useful, and competitive platform for XR.

I've attempted to distill the items we talked about in the last CG call into bullet points and to slot them into the years when we'll see them start to really take off.

Please leave comments in this PR or email me (trevor@transmutable.com) so that we can create a useful document for the many conversations we all have about the future of the immersive web.

### CODE REVIEW

File: Tech Radar 2019.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- shared anchors
+- long lived, simultanous XR apps
+- deeper OS integration
+
+2022:
+
+- shared layers / spaces
+- delarative content
+- personal agents
+- page-defined computer vision
```
Comment (blairmacintyre) [inline Tech Radar 2019.md]: 2022 for CV?  That seems somewhat far out.  We should have CV in pages next year, I think.  It's pretty straightforward.

File: Tech Radar 2019.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+2020:
+
+- payment integration
+- hand gesture input
+- eye tracking
+
+2021:
+
+- shared anchors
+- long lived, simultanous XR apps
```
Comment (blairmacintyre) [inline Tech Radar 2019.md]: This seems too soon:  long lived, simultaneous apps imply things for the browser UIs (lots of things) which nobody is even thinking about yet.  2022 or longer, I'd say.

File: Tech Radar 2019.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+- deeper OS integration
+
+2022:
+
+- shared layers / spaces
+- delarative content
+- personal agents
+- page-defined computer vision
+- portable avatars
```
Comment (blairmacintyre) [inline Tech Radar 2019.md]: Without context, this is "meaningless" I think.  Avatars for what?  How are they rendered?  How are they included in a page / app?  Or are they supported by the browser?  We could do a "gravatar3d" tomorrow, but who would use it and how?

Comment (AlbertoElias) [reply Tech Radar 2019.md]: Yeah, I'm doing something like this with Simbol (library for content creators to use), but that's far from WebXR native browser extensions or proper browser implementation.

**Pull request closed without merging**